masonry/src/widget/widget_state.rs

```diff
@@ -5,11 +5,17 @@
 
 use std::sync::atomic::{AtomicBool, Ordering};
 
+use kurbo::Vec2;
```

Comment:
`use crate::kurbo::Vec2;`

Reply:
Actually, can combine with the import below.

Reply:
(I mean it's just a nit, but I agree, it's *slightly* cleaner, same for the other `use` comments here)

Reply:
Won't fix, see other comment.

Reply:
We should probably generally find a unique style in the future, just to be consistent.

Reply:
We've had an agreement in the past in #223, but I needed to rebase it and it slipped through the cracks.

I've resurrected it.